### PR TITLE
docs: .git-blame-ignore-revs

### PR DESCRIPTION
**Pull request by justinmk**
Will add a few more, before merging this.

### CODE REVIEW

**Comment by dundargoc** [comment]
You can add commit 61178778230e609d68b271ffd53ffd993cd23c42 to the uncrustify list as well.

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -7,5 +7,8 @@
 #     save you the trouble of attempting this.
 6c5bbf07d988ef55e5e8ba8d70b62c1f0885261b
 
+# symbol renames
+6186df3562e33e92f04ed8c850204ceabc4746e1
+
```
Comment (justinmk) [inline .git-blame-ignore-revs:12]:
```suggestion

61178778230e609d68b271ffd53ffd993cd23c42
15af08ad176339d1f269ce264bb0efea283c9536
47f99d66440ae8be26b34531989ac61edc1ad9fe
1e49a1c888a3d9a581f4aa409a26ada3ac2417cb
3b3dbcf7b7ba5466e6ab643e256f2374b520a6b2
e8067d1490a31ff76143d576dc9948b4f09c6c55
```

File: .git-blame-ignore-revs [hunk 2]
```diff
@@ -7,5 +7,14 @@
...
+61178778230e609d68b271ffd53ffd993cd23c42
+15af08ad176339d1f269ce264bb0efea283c9536
+47f99d66440ae8be26b34531989ac61edc1ad9fe
+1e49a1c888a3d9a581f4aa409a26ada3ac2417cb
+3b3dbcf7b7ba5466e6ab643e256f2374b520a6b2
+e8067d1490a31ff76143d576dc9948b4f09c6c55
```
Comment (justinmk) [inline .git-blame-ignore-revs:18]:
```suggestion
e8067d1490a31ff76143d576dc9948b4f09c6c55

```

File: .git-blame-ignore-revs [hunk 3]
```diff
@@ -7,5 +7,14 @@
 #     save you the trouble of attempting this.
 6c5bbf07d988ef55e5e8ba8d70b62c1f0885261b
 
+# symbol renames
+6186df3562e33e92f04ed8c850204ceabc4746e1
+
```
Comment (justinmk) [inline .git-blame-ignore-revs:12]:
```suggestion
```

**Pull request merged**